recipes/srwpy/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "srwpy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Synchrotron-Radiation-Workshop/SRW/archive/v{{ version }}.tar.gz
+  sha256: 53bdcb0c02c59659bd0cb3849eac15c166db65b69d6fb62d4b8bf97cc4c96be8
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - fftw  # [win]
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - fftw  # [win]
+    - matplotlib-base
+    - numpy
+    - pillow
+    - scipy
+
+test:
+  imports:
+    - srwpy
+    - srwpy.srwl_bl
+    - srwpy.srwlib
+    - srwpy.srwlpy
+    - srwpy.uti_math
+
+about:
+  home: https://github.com/Synchrotron-Radiation-Workshop/SRW
+  license: LicenseRef-SRW
+  license_file: LICENSE
```

Do the following and delete `recipe/LICENSE`
```suggestion
  license_file: COPYRIGHT.txt
```

---

Addressed via https://github.com/conda-forge/staged-recipes/pull/14021/commits/7ac05f1c4876fd14819c7b44898c68f1b2bcc207